lms/djangoapps/certificates/views/webview.py

```diff
@@ -149,8 +149,7 @@ def _update_certificate_context(context, course, user_certificate, platform_name
                                                 u"certificates, which are awarded for course activities "
                                                 u"that {platform_name} students complete.").format(
         platform_name=platform_name,
-        tos_url=context.get('company_tos_url'),
-        verified_cert_url=context.get('company_verified_certificate_url'))
+    )
```

Comment:
This file is the only non-test change - and I just wanted to confirm that the above is intentional. Without context, it doesn't look microsite specific to me. But you have all the context.

Reply:
Ah. right.  I removed these 2 lines of dead code since these parameters are not used by the preceding string template.